### PR TITLE
Website: add an all.xml rss feed

### LINKED ISSUE #6068: Expose more EIP info via RSS
### Proposed Change

It's been requested that we have better hooks for interested parties to view EIP activity. Can we start with ERC category EIPs entering different stages.

### PR DESCRIPTION
**Pull request by moodysalem**
fixes https://github.com/ethereum/EIPs/issues/6068

### CODE REVIEW

File: all.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    <description>A feed of all EIPs</description>
+    <link>{{ site.url }}</link>
+    <atom:link href="{{ site.url }}/all.xml" rel="self" type="application/rss+xml" />
+    <lastBuildDate>{{ site.time | date_to_rfc822 }}</lastBuildDate>
+    {% assign eips = site.pages | sort: 'eip' %}
+    {% for eip in eips %}
+      <item>
+        <title>{{ eip.title | xml_escape }}</title>
+        <category>{{ eip.type }}/{{ eip.category }}</category>
+        <author>{{ eip.author | xml_escape }}</author>
```
Comment (SamWilsn) [inline all.xml:17]: I think `<author>` needs to be an email address.

File: all.xml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        {% if eip.discussions-to %}
+          <comments>{{ eip.discussions-to | xml_escape }}</comments>
+        {% else %}
```
Comment (SamWilsn) [inline all.xml:20]:
```suggestion
        {% endif %}
```

**Review by Pandapip1 — CHANGES_REQUESTED** [review]
I would definitely like to move all of the rss stuff into an `rss` folder.

**Comment by moodysalem** [comment]
Moved it, not sure what else is needed

**Review by Pandapip1 — APPROVED** [review]

**Review by SamWilsn — APPROVED** [review]

**Pull request merged**